src/components/Header/HeaderContact.js

```diff
@@ -6,6 +6,7 @@ import {fonts, normalizeFontSize} from '../../utils/fonts';
 import {COLORS, SIZES} from '../../utils/theme';
 import GlobalButton from '../Button/GlobalButton';
 import dimen from '../../utils/dimen';
+import {colors} from '../../utils/colors';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `colors` is added but the old import for `COLORS` is not removed. This could lead to confusion and potential bugs in the future if someone tries to use `COLORS` instead of `colors`. It's better to remove unused imports to keep the code clean and maintainable.
```diff
- import {COLORS, SIZES} from '../../utils/theme';
+ import {SIZES} from '../../utils/theme';
  import GlobalButton from '../Button/GlobalButton';
  import dimen from '../../utils/dimen';
  import {colors} from '../../utils/colors';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->